client/src/js/components/bhAddItem.js

```diff
@@ -0,0 +1,35 @@
+angular.module('bhima.components')
+  .component('bhAddItem', {
+    templateUrl : 'modules/templates/bhAddItem.tmpl.html',
+    controller  : addItemController,
+    transclude  : true,
+    bindings    : {
+      itemIncrement    : '<',
+      recipient        : '<',
+      disable          : '<?',
+      onChange         : '&?',
+      name             : '@?',
+    },
+  });
+
+addItemController.$inject = [];
+
+/**
+ * Add Item component
+ *
+ */
+function addItemController() {
+  var $ctrl = this;
+
+  $ctrl.$onInit = function onInit() {    
+    // default for form name
+    $ctrl.name = $ctrl.name || 'AddItemForm';
+  };
+
+  // fires the onSelectCallback bound to the component boundary
```

Comment:
I don't think there is a `onSelectCallback`.  Could you remove this comment or update it?